packages/dataviews/src/view-table.js

```diff
@@ -171,17 +175,9 @@ function HeaderMenu( { field, view, onChangeView } ) {
 							<WithSeparators>
 								<DropdownMenuGroup>
 									{ filter.elements.map( ( element ) => {
-										let isActive = false;
-										if ( filterInView ) {
-											// Intentionally use loose comparison, so it does type conversion.
```

Comment:
I could no longer reproduce this issue, so this changed to strict comparison like the other places.